chart/templates/herald-config.yaml

```
@@ -4,7 +4,7 @@ metadata:
   name: {{ .Chart.Name }}
   namespace: {{ .Values.namespace }}
 data:
-  herald-config.yaml: |
+  herald-config.yaml:
     {{- with .Values.heraldConfig }}
     {{- toYaml . | nindent 4 }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)